resources/prometheus/federation-config.yaml

```
@@ -30,6 +30,9 @@
       - apiserver_request:burnrate6h{job!~"central|scanner"}
       - apiserver_request_terminations_total{job!~"central|scanner"}
       - apiserver_request_total{job!~"central|scanner"}
+      - argocd_app_info{job!~"central|scanner"}
+      - argocd_app_sync_total{job!~"central|scanner"}
+      - argocd_notifications_deliveries_total{job!~"central|scanner"}
```

I don't think the filter makes sense, it has to do with the metrics exposed by central/scanner itself.

```suggestion
      - argocd_app_info
      - argocd_app_sync_total
      - argocd_notifications_deliveries_total
```

---

This file is generated :/